libs/chain-signatures/contract/src/tee/tee_state.rs

```diff
@@ -15,9 +12,25 @@ use mpc_primitives::hash::LauncherDockerComposeHash;
 use near_sdk::{env, near, store::IterableMap, AccountId, PublicKey};
 use std::collections::HashSet;
 
-pub enum TeeValidationResult {
+pub(crate) enum TeeValidationResult {
+    /// All participants have a valid attestation.
     Full,
-    Partial(Participants),
+    /// Only a subset of the participants had a valid attestation.
+    Partial {
+        participants_with_valid_attestation: Participants,
+    },
+}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub(crate) enum TeeQuoteStatus {
+    /// TEE quote and Docker image verification both passed successfully.
+    /// The participant is considered to have a valid, verified TEE status.
+    Valid,
+
+    /// TEE verification failed - either the quote verification failed,
+    /// the Docker image verification failed, or both validations failed.
+    /// The participant should not be trusted for TEE-dependent operations.
+    Invalid,
```

Comment:
I know you just copy-pasted the comments, but I think they’re not precise since quote verification involves many more checks. For example, RTMRs can be invalid, firmware/certs can be outdated, and there are several other cases as well.